salt/metalk8s/kubernetes/kubelet/standalone.sls

```diff
@@ -70,7 +70,13 @@ Create kubelet config file:
         httpCheckFrequency: 0s
         imageMinimumGCAge: 0s
         kind: KubeletConfiguration
-        logging: {}
+        logging:
+          flushFrequency: 0
+          options:
+            json:
+              infoBufferdockeSize: "0"
```

Comment:
Is this variable name correct? Looks like a typo, but I guess you just copy-pasted the kubeadm manifests?

Reply:
Right it's "infoBufferSize" (likely tried to hit docker in a wrong tab :smile: )